include/crypto/aes_platform.h

```diff
@@ -65,16 +65,16 @@ void AES_xts_decrypt(const unsigned char *inp, unsigned char *out, size_t len,
 #   ifdef VPAES_ASM
 #    define VPAES_CAPABLE (OPENSSL_ppccap_P & PPC_ALTIVEC)
 #   endif
-#   define HWAES_CAPABLE  (OPENSSL_ppccap_P & PPC_CRYPTO207)
-#   define HWAES_set_encrypt_key aes_p8_set_encrypt_key
-#   define HWAES_set_decrypt_key aes_p8_set_decrypt_key
-#   define HWAES_encrypt aes_p8_encrypt
-#   define HWAES_decrypt aes_p8_decrypt
-#   define HWAES_cbc_encrypt aes_p8_cbc_encrypt
-#   define HWAES_ctr32_encrypt_blocks aes_p8_ctr32_encrypt_blocks
-#   define HWAES_xts_encrypt aes_p8_xts_encrypt
-#   define HWAES_xts_decrypt aes_p8_xts_decrypt
 #   if !defined(OPENSSL_SYS_AIX) && !defined(OPENSSL_SYS_MACOSX)
```

Comment:
IIRC some Darwin systems were 64 bit PPC.  Is this going to disable the asm support for those platforms?  Or is that acceptable here?

Reply:
No, it won't disable the asm support for AES completely. It's just the PowerISA 2.07 code that won't be built. And AFAIK it was not used anyway.

Reply:
PowerISA 2.07 code won't be built for 32-bit and 64-bit; so _that_ code is gone from macOS regardless of whether its supported (i.e. on 32-bit)?

Reply:
@tmshort AFAIK it was never supported by the CPUs used on Macs? Its just that the assembler previously built but it was not used anyway.

Reply:
G5 is ISA 2.02 with some extensions, AFAIK. Anything above ISA 2.03 is off question for sure.